File: 0034.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
	Scanner sc = new Scanner(System.in);
	while(sc.hasNext())
	{
	    String line = sc.next();
	    String[] s = line.split(",");
	    int sum = 0;
	    for(String v : s)
		sum += Integer.parseInt(v);
	    int l = 0;
	    for(int i = 0; i < s.length; i++)
	    {
		l += Integer.parseInt(s[i]);
		if(l >= sum /)
		{
		    System.out.println(i + 1);
		    break;
		}
	    }
	}
    }
}

File: 0034.py
import sys

for line in sys.stdin:
    ll = list(map(int, line.split(',')))
    s = ll[:10]
    v1, v2 = ll[10], ll[11]
    p = sum(s) / (v1 + v2) * v1

    ss = 0
    for i, l in enumerate(s):
        ss += l
        if ss >= p:
            print(i + 1)
            break
